entrypoint.sh

```diff
@@ -100,37 +99,126 @@ function super_lint() {
   /action/lib/linter.sh
 }
 
+function get_files_matching_filetype() {
+  local matching_files=()
+  local filenames=("${@:2}")
+  local filetype=$1
```

Comment:
I think a better way to do this and the prior line would be to replace it with this so we don't need to slice the array:

```suggestion
  local filetype=$1
  shift
  local filenames=("$@")
```